openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -518,6 +527,41 @@ def test_course_home_for_global_staff(self, get_patched_current_site):
         content = pq(response.content)
         self.assertTrue(content('.action-resume-course').attr('href').endswith('/course/' + course.url_name))
 
+    @override_switch(
+        '{}.{}'.format(
+            waffle.WAFFLE_NAMESPACE, waffle.ENABLE_COMPLETION_TRACKING
+        ),
+        active=True
+    )
+    @patch('completion.waffle.get_current_site')
+    def test_course_outline_auto_open(self, get_patched_current_site):
```

Comment:
Looks like you are missing an pylint ignore comment here for the unused argument.